src/bindings.zig

```diff
@@ -0,0 +1,52 @@
+const std = @import("std");
```

Comment:
Two thoughts for improvement:

1. Should these bindings live here, or in `libmach` I wonder? The same question would go for other Mach libraries in the future, e.g. when we expose a C ABI for the ECS library, I wonder where it should live?

2. We should plan on changing this API to reflect what Ayush said in Matrix, something like this as our long-term goal (but may require some changes to make this possible):

```
int main(void) {
    mach_platform_init(init_fn, usr_ptr);
    while (...) {
        mach_platform_update(update_fn, resize_fn, usr_ptr);
    }
    mach_platform_deinit(deinit_fn, usr_ptr);
}
```

Reply:
My opinion on (1) I think the bindings should live inside src/ maybe changing the name to libmach.zig to be more clear. The libmach directory currently doesn't serves any purpose, it just contains examples.